backend/src/main/java/fk/prof/backend/util/ZookeeperUtil.java

```diff
@@ -50,12 +61,11 @@ public static void writeZNode(CuratorFramework curatorClient, String zNodePath,
     return future;
   }
 
-  //TODO: Keeping this around in case required for policy CRUD. If not used there, remove
-  public static Future<Void> writeZNodeAsync(CuratorFramework curatorClient, String zNodePath, byte[] data, boolean create)
+  public static Future<Void> writeZNodeAsync(CuratorFramework curatorClient, String zNodePath, byte[] data, boolean create, CreateMode mode)
```

Comment:
Add a comment here, explaining that this method creates parents if missing. This is not apparent from function signature.